scripts/index_d_ts

```diff
@@ -6,7 +6,7 @@
 declare module "SendbirdUIKitGlobal" {
   import type React from 'react';
   import type SendbirdChat from '@sendbird/chat';
-  import { GroupChannelModule, ModuleNamespaces, OpenChannelModule } from '@sendbird/chat/lib/__definition';
+  import { GroupChannel, GroupChannelModule, ModuleNamespaces, OpenChannel, OpenChannelModule } from '@sendbird/chat/lib/__definition';
```

Comment:
can remove this

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/613/commits/e10c8818bdf9c6b958deacdc6a22116246dd81f4